src/routes/createNewHouse.routes.js

```diff
@@ -0,0 +1,14 @@
+import createNewHouseController from '../controllers/createNewHouse.controllers';
+
+const createNewHouseRoute = (router) => {
+    router.post('/create-new-house', async (req, res) => {
+      const newHouse = await createNewHouseController(req.body)
+      try {
+          res.status(201).json(newHouse)
+      } catch (err){
+          res.status(400).json({ message: err.message})
```

Comment:
The rest of us use special packages that are inserting statuses while using more readable names for it.
`const { StatusCodes }= require( 'http-status-codes');`